server/src/packages/packages.service.ts

```diff
@@ -196,7 +196,9 @@ export class PackagesService {
     } catch (e) {
       // relay lower-level exceptions, like from crmServce.get(),
       // or sharepoint document retrieval.
+      console.log('dcp_project.artifact error', e);
```

Comment:
this looks like an inaccurate message. The try/catch here wraps the entire procedure to retrieve a package